books/free-programming-books-es.md

```diff
@@ -238,6 +239,11 @@
 * [SurviveJS - React de aprendiz a maestro](https://es.survivejs.com) - Juho Vepsäläinen, `trl.:` Raúl Expósito (HTML, PDF)
 
 
+### Kotlin
+
+* [Curso programación Android en Kotlin](https://cursokotlin.com/curso-programacion-kotlin-android/) - AristiDevs (HTML)
```

Comment:
Don't invent titles! Use the title given by the resource.

Reply:
Which title should I choose? I mean:

- The main head of the URL is "Curso Kotlin para Android", but that is the title of the whole web page, which also includes another book for "Jetpack Compose". Clicking on it leads to the main page.
- It looks like "Curso programación Android en Kotlin" is the actual name for this particular book since that is the title of the URL. Also, on the main page, the link to access this particular book and not the other one is also "Curso programación Android en Kotlin".

It looks to me that the initial plan was to have only the Kotlin tutorial, and that is why the domain and title of the page are confusing.

Reply:
> Don't invent titles! Use the title given by the resource.

I still think that "Curso programación Android en Kotlin" is the actual name, but I will change it if you think otherwise.